test/ui-test/testSuites/suite_status/tst_createCommunityChannel/test.feature

```diff
@@ -0,0 +1,32 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file	test.feature
+# *
+# * \test	Status Desktop - Community Channel
+# * \date	July 2022
+# **
+# *****************************************************************************/
+
+Feature: Status Desktop community
```

Comment:
Why do you think it is a new test case? Maybe we can add it into the `tst_createCommunity` as a new scenario. The background is the same until the user lands on the community portal section and then, there are some specific scenarios we test related to communities. WDYT?

Maybe we should rename the `tst_createCommunity` to something more "community generic"?  Like.. `tst_communityFlows`??